templates/pioneer-study/esper-study-telemetry.1.parquetmr.txt

```diff
@@ -0,0 +1,73 @@
+message esper-study-telemetry {
+  required group metadata {
+    required int64  Timestamp;
+    required binary pioneerId (UTF8);
+    required binary studyName (UTF8);
+    optional binary reason (UTF8);
+    optional binary Date  (UTF8);
+    optional binary DNT (UTF8);
+    optional binary appUpdateChannel (UTF8);
+    required binary documentId (UTF8);
+    optional binary docType (UTF8);
+    optional binary os (UTF8);
+    optional binary environment.addons (UTF8);
```

Comment:
This field does not apply to shield-related pings and should be removed. Similarly `environment.partner`, `environment.profile`, `environment.settings`, `environment.system`, `environment.build`, `environment.experiments`.

Reply:
Thanks, removed